onadata/libs/utils/async_status.py

```diff
@@ -3,14 +3,18 @@
 PENDING = 0
 SUCCESSFUL = 1
 FAILED = 2
+PROGRESS = 3
+RETRY = 4
+STARTED = 5
```

Comment:
that makes sense, I am asking about the difference between, `PROGRESS` and `STARTED`